devices/atlantic.js

```diff
@@ -0,0 +1,127 @@
+const fz = require('../converters/fromZigbee');
+const tz = require('../converters/toZigbee');
+const exposes = require('../lib/exposes');
+const reporting = require('../lib/reporting');
+const utils = require('../lib/utils');
+const assert = require('assert');
+const e = exposes.presets;
+const ea = exposes.access;
+
+
+const thermostatPositions = {
+    'barley_closed': 0,
+    'quarter_open': 1,
+    'half_open': 2,
+    'three_quarters_open': 3,
+    'fully_open': 4,
+    'swing': 5,
+};
+
+const energyMode = {
+    'eco': 0,
+    'normal': 1,
+    'powerfull': 2,
+};
+
+const tzLocal = {
+    quiet_fan: {
+        key: ['quiet_fan'],
+        convertSet: async (entity, key, value, meta) => {
+            assert(typeof value === 'boolean');
+            // todo: failed (Status 'UNSUPPORTED_ATTRIBUTE')
+            await entity.write('hvacFanCtrl', {0x1000: {value: 1, type: 0x10}});
+            return {state: {quiet_fan: value}};
+        },
+    },
+    ac_louver_position: {
+        key: ['ac_louver_position'],
+        convertSet: async (entity, key, value, meta) => {
+            value = value.toLowerCase();
+            utils.validateValue(value, Object.keys(thermostatPositions));
+            const index = thermostatPositions[value];
+            if (index === 5) {
+                // todo: failed (Status 'UNSUPPORTED_ATTRIBUTE')
+                await entity.write('hvacFanCtrl', {0x4274: {value: 1, type: 0x10}});
```

Comment:
There are 5 positions for ac louver position: open from 1 to 4, and swing.
For swing:

From Hub:
```
Frame 2757: 54 bytes on wire (432 bits), 52 bytes captured (416 bits) on interface [\\.\pipe\zboss_sniffer_COM7](file:////pipe/zboss_sniffer_COM7), id 0
IEEE 802.15.4 Data, Dst: 0x0689, Src: 0x0000
ZigBee Network Layer Data, Dst: 0x0689, Src: 0x0000
ZigBee Application Support Layer Data, Dst Endpt: 1, Src Endpt: 1
    Frame Control Field: Data (0x40)
    Destination Endpoint: 1
    Cluster: Thermostat (0x0201)
    Profile: Home Automation (0x0104)
    Source Endpoint: 1
    Counter: 25
ZigBee Cluster Library Frame, Mfr: Atlantic Group (0x125b), Command: Write Attributes, Seq: 61
    Frame Control Field: Profile-wide (0x04)
    Manufacturer Code: Atlantic Group (0x125b)
    Sequence Number: 61
    Command: Write Attributes (0x02)
    Attribute Field, Boolean: 0x01
        Attribute: 0x4274
        Data Type: Boolean (0x10)
        0000 0001 = Boolean: True
```

From zigbee2mqtt with my converter:
```
Frame 3: 52 bytes on wire (416 bits), 50 bytes captured (400 bits) on interface [\\.\pipe\zboss_sniffer_COM7](file:////pipe/zboss_sniffer_COM7), id 0
IEEE 802.15.4 Data, Dst: 0x0cc1, Src: 0x0000
ZigBee Network Layer Data, Dst: 0x0cc1, Src: 0x0000
ZigBee Application Support Layer Data, Dst Endpt: 1, Src Endpt: 1
    Frame Control Field: Data (0x00)
    Destination Endpoint: 1
    Cluster: Fan Control (0x0202)
    Profile: Home Automation (0x0104)
    Source Endpoint: 1
    Counter: 219
ZigBee Cluster Library Frame, Command: Write Attributes, Seq: 85
    Frame Control Field: Profile-wide (0x10)
    Sequence Number: 85
    Command: Write Attributes (0x02)
    Attribute Field, Boolean: 0x01
        Attribute: Unknown (0x4274)
        Data Type: Boolean (0x10)
        0000 0001 = Boolean: True
```

Reply:
should be
```javascript
await entity.write('hvacFanCtrl', {0x1000: {value: value ? 1 : 0, type: 0x10}});
```